cmd/notifier.go

```diff
@@ -0,0 +1,100 @@
+package cmd
+
+import (
+	"finala/notifiers"
+	notifiersCommon "finala/notifiers/common"
+	"finala/notifiers/config"
+	"finala/request"
+	"finala/visibility"
+	"fmt"
+	"os"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+const (
+	notifierCommandName = "notifier"
+)
+
+// collectorCMD will present the aws analyze command
+var notifierCMD = &cobra.Command{
+	Use:   notifierCommandName,
+	Short: "Notify set of users/emails from a given configuration",
+	Long:  ``,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		notifierLog := log.WithField("command_name", notifierCommandName)
```

Comment:
Once you decide to wrap the logger with generic field (this is the best practice), you should make sure that all the logs that notifier using the log

Reply:
Done.